assets/js/lib/network/index.js

```diff
@@ -0,0 +1,86 @@
+import axios from 'axios';
+import { logError, logWarn } from '@lib/log';
+
+const conf = {
+  validateStatus: (status) => {
+    return status < 500;
+  },
+};
+
+export const axiosPost = function (url, data) {
+  return new Promise((resolve, reject) => {
+    axios
+      .post(url, data, conf)
+      .then((response) => {
```

Comment:
I think you can avoid returning a new promise because `axios.post` already returns a promise, so basically you can `return axios.post(...).then(handleResponseStatus)`.

What do you think?

Reply:
Sure, returning a new promise is a bit more explicit but it does indeed add unneeded code